audit.go

```diff
@@ -34,11 +36,11 @@ func MakeAuditor(num int, pki *PKI) *Auditor {
 		pki:          pki,
 		Setup:        make(chan struct{}),
 	}
-	a.CommsCache = make([]ECPoint, a.num)
-	a.RTokenCache = make([]ECPoint, a.num)
+	a.CommsCache = make([]zksigma.ECPoint, a.num)
+	a.RTokenCache = make([]zksigma.ECPoint, a.num)
 	for i := 0; i < a.num; i++ {
-		a.CommsCache[i] = EC.Zero()
-		a.RTokenCache[i] = EC.Zero()
+		a.CommsCache[i] = zksigma.Zero
```

Comment:
how is this safe?  it means all the caches are using the same underlying big ints, right?

Reply:
As far as I can tell, the only operation done on these cache points is Add. And an Add to Zero will return the point you're adding. But even an Add between two non-zero points returns new BigInts. So the Zero point will never be modified.